docs/src/main/sphinx/release/release-435.md

```diff
@@ -0,0 +1,105 @@
+# Release 435 (6 Dec 2023)
+
+## General
+
+* Fix possible query failure for `MERGE` queries when `retry-policy` set to
+  `TASK` and `query.determine-partition-count-for-write-enabled` set to `true`.
+  ({issue}`19979`)
+
+## Security
+
+## Web UI
+
+## JDBC driver
+
+## Server RPM
+
+## Docker image
+
+## CLI
+
+## Accumulo connector
+
+## Atop connector
+
+## BigQuery connector
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+## Delta Lake connector
+
+# Delta Lake
+
+* {{breaking}} Remove support for registering external tables with
```

Comment:
fyi @marcinsbd @ebyhr .. here is the reworded entry

Reply:
The table property name should be `location` instead of `external_location`. Otherwise, looks good to me.

Reply:
Just noticed myself .. fixed.